README.md

```diff
@@ -166,6 +166,19 @@ Run the go tests to test:
 $ go test ./...
 ```
 
+#### DB Migration end-to-end tests
```

Comment:
It'd be nice if this could be automated (even partially) however I think just putting instructions in the README is the right first step, especially because this kind of thing can be laborious to automate with (in my opinion) not much value return. I intend to follow your lead for other e2e tests in this repo, as well as for the AWAT.

But maybe it makes sense to open a JIRA ticket to represent the work of automating setup for e2e tests at some point? I'm 0/5 on that suggestion, but I thought it was worth throwing it out there anyway.

Reply:
Yeah, automatic full setup at this point would be overkill at this point. I think it can be part of some more wholesome e2e test, also doing the setup could be much easier with terraform, etc. than doing it here, especially we would need some cleanup also.

I will create a more broad ticket for incorporating DB migration tests to some e2e flow.